mkdocs.yml

```diff
@@ -61,13 +60,14 @@ nav:
             - Configuring private Services: developer/serving/services/private-services.md
             - Configuring custom domains: developer/serving/services/custom-domains.md
             - Configure resource requests and limits: developer/serving/services/configure-requests-limits-services.md
+            - Modifying the Deployment Config Map: developer/serving/services/deployment.md
```

Comment:
I moved this to the admin guide, think it belongs there?

Reply:
i.e. I'd remove this entirely and just leave it in the admin guide or we're about to have two...

Reply:
ohhhhhh that was the merge conflict

Reply:
ok so I can remove this and it'll still be in the admin guide?

Reply:
Yep 😄 it's here https://deploy-preview-3728--dev-knative.netlify.app/development/admin/serving/deployment/ so you can remove it from anywhere else

Reply:
Don't forget to delete any files as well as taking it out of the yml 😉

Reply:
@omerbensaadon don't forgetttttttt to take it out of the yml